Plugins/Flow.Launcher.Plugin.Program/Programs/UWP.cs

```diff
@@ -226,10 +233,31 @@ private static IEnumerable<Package> CurrentUserPackages()
             }
             else
             {
-                return new Package[] { };
+                return new Package[]
+                {
+                };
             }
         }
 
+        private static List<FileSystemWatcher> _watchers = new();
+
+        private static void GenerateWatcher(string path)
+        {
+            var watcher = new FileSystemWatcher(path);
+            watcher.Created += static (_, _) => Task.Run(Main.IndexUwpPrograms);
+            watcher.Deleted += static (_, _) => Task.Run(Main.IndexUwpPrograms);
+            watcher.EnableRaisingEvents = true;
+        }
+
+        public static void WatchPackageChange()
+        {
+            if (Environment.OSVersion.Version.Build >= 19041)
```

Comment:
why is this condition needed?

Reply:
Same to uwp apps. Probably using main version higher than 10 is also good.

Reply:
sorry i am not following, main version as in Windows 10? why higher than 10 is good though?